crates/aptos/src/governance/mod.rs

```diff
@@ -160,9 +160,13 @@ pub struct SubmitVote {
     #[clap(long)]
     pub(crate) proposal_id: u64,
 
-    /// Vote choice. True for yes. False for no.
-    #[clap(long)]
-    pub(crate) should_pass: bool,
+    /// Vote yes on the proposal
+    #[clap(long, group = "vote")]
+    pub(crate) yes: bool,
+
+    /// Vote no on the proposal
+    #[clap(long, group = "vote")]
```

Comment:
Let's add `conflicts_with`

Reply:
Or is this what `group` is for?

Reply:
group yeah only lets you have one or the other